### PR TITLE
[zh-cn] sync container-runtimes configmap

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/concepts/configuration/configmap.md
content/zh-cn/docs/setup/production-environment/container-runtimes.md

### CODE REVIEW

File: content/zh-cn/docs/concepts/configuration/configmap.md [hunk 1]
```diff
@@ -363,6 +363,59 @@ A container using a ConfigMap as a [subPath](/docs/concepts/storage/volumes#usin
 使用 ConfigMap 作为 [subPath](/zh-cn/docs/concepts/storage/volumes#using-subpath) 卷挂载的容器将不会收到 ConfigMap 的更新。
 {{< /note >}}
 
+<!--
+### Using Configmaps as environment variables
+
+To use a Configmap in an {{< glossary_tooltip text="environment variable" term_id="container-env-variables" >}}
+in a Pod:
+-->
+### 使用 Configmap 作为环境变量
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/configuration/configmap.md:372]:
```suggestion
### 使用 Configmap 作为环境变量  {#using-configmaps-as-environment-variables}
```

File: content/zh-cn/docs/concepts/configuration/configmap.md [hunk 2]
```diff
@@ -363,6 +363,59 @@ A container using a ConfigMap as a [subPath](/docs/concepts/storage/volumes#usin
...
+使用 Configmap 在 Pod 中设置{{< glossary_tooltip text="环境变量" term_id="container-env-variables" >}}：
+
+<!--
+1. For each container in your Pod specification, add an environment variable
+   for each Configmap key that you want to use to the
+   `env[].valueFrom.configMapKeyRef` field.
+1. Modify your image and/or command line so that the program looks for values
+   in the specified environment variables.
+-->
+1. 对于 Pod 规范中的每个容器，为要使用的每个 Configmap 键添加一个环境变量到
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/configuration/configmap.md:383]:
```suggestion
1. 对于 Pod 规约中的每个容器，为要使用的每个 ConfigMap 键添加一个环境变量到
```

File: content/zh-cn/docs/concepts/configuration/configmap.md [hunk 3]
```diff
@@ -363,6 +363,59 @@ A container using a ConfigMap as a [subPath](/docs/concepts/storage/volumes#usin
...
+   `env[].valueFrom.configMapKeyRef` 字段。
+2. 修改你的镜像 和/或 命令行，以便程序查找指定环境变量中的值。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/configuration/configmap.md:385]:
```suggestion
2. 修改你的镜像和/或命令行，以便程序查找指定环境变量中的值。
```

File: content/zh-cn/docs/concepts/configuration/configmap.md [hunk 4]
```diff
@@ -363,6 +363,59 @@ A container using a ConfigMap as a [subPath](/docs/concepts/storage/volumes#usin
...
+
+<!--
+This is an example of defining a ConfigMap as a pod environment variable:
+-->
+这是将 ConfigMap 定义为 Pod 环境变量的示例：
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/configuration/configmap.md:390]:
```suggestion
下面是一个将 ConfigMap 定义为 Pod 环境变量的示例：
```

File: content/zh-cn/docs/concepts/configuration/configmap.md [hunk 5]
```diff
@@ -363,6 +363,59 @@ A container using a ConfigMap as a [subPath](/docs/concepts/storage/volumes#usin
...
+```
+
+<!--
+It's important to note that the range of characters allowed for environment
+variable names in pods is [restricted](/docs/tasks/inject-data-application/define-environment-variable-container/#using-environment-variables-inside-of-your-config).
+If any keys do not meet the rules, those keys are not made available to your container, though
+the Pod is allowed to start.
+-->
+需要注意的是，Pod 中环境变量名称允许的字符范围是[有限的](/zh-cn/docs/tasks/inject-data-application/define-environment-variable-container/#using-environment-variables-inside-of-your-config)。
+如果任何 key 不符合规则，则尽管允许 Pod 启动，但这些 key 将无法供你的容器使用。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/configuration/configmap.md:417]:
```suggestion
如果某些变量名称不满足这些规则，则即使 Pod 可以被启动，你的容器也无法访问这些环境变量。
```

**Review by ydFu — APPROVED** [review]
/approve

**Pull request merged**